.github/workflows/source_aws.yml

```diff
@@ -153,8 +154,7 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version-file: plugins/source/aws/go.mod
-          cache: true
-          cache-dependency-path: plugins/source/aws/go.sum
+          cache: false
```

Comment:
```suggestion
```

This already the default so not needed. We can do it once we update to v4

Reply:
I think it's better to leave this explicitly turned off as it doesn't affect the result now

Reply:
What's the added value of setting a configuration to the default value?